ui-v2/tests/helpers/api.js

```diff
@@ -27,6 +27,7 @@ export const get = function(_url, options = { headers: { cookie: {} } }) {
         }, {}),
       },
       {
+        set: function() {},
         send: function(content) {
```

Comment:
This looks weird I know! 😄

I noticed that when my api-double tries to send headers when its being run as a HTTP server it calls this function, everything is fine when its running over HTTP, but when I'm running the double in the HTML page without the server it looks to call this if there are headers to be sent. Right now I have no tests that are testing what comes back from consul, and I not sure if I ever will, so for now at least this is a noop when used in the frontend/without a server.